docs/concepts/runtime.md

```diff
@@ -46,19 +46,23 @@ The Flink runtime consists of two types of processes:
   - The **JobManagers** (also called *masters*) coordinate the distributed execution. They schedule tasks, coordinate
     checkpoints, coordinate recovery on failures, etc.
 
-    There is always at least one Job Manager. A high-availability setup will have multiple JobManagers, one of
-    which one is always the *leader*, and the others are *standby*.
+    There is always at least one Job Manager. A high-availability setup should have multiple JobManagers, one of
```

Comment:
"JobManager"?

Reply:
"A high-availability setup should have multiple JobManagers" is, in general, not true -- this is a detail that depends on the underlying cluster management framework. 

I suggest reworking as follows:

There is always at least one Job Manager, but some [high-availability setups]({{ site.baseurl }}/ops/jobmanager_high_availability.html) will have multiple JobManagers.